packages/voting-contract/src/lib.rs

```diff
@@ -413,9 +413,9 @@ pub fn list_votes_by_voter(
 pub fn query_voter(deps: Deps, voter: String) -> StdResult<VoterResponse> {
     let cfg = CONFIG.load(deps.storage)?;
     let voter_addr = deps.api.addr_validate(&voter)?;
-    let weight = cfg.group_contract.is_member(&deps.querier, &voter_addr)?;
+    let points = cfg.group_contract.is_member(&deps.querier, &voter_addr)?;
 
-    Ok(VoterResponse { weight })
+    Ok(VoterResponse { points })
```

Comment:
Or is this only used in voting-contract and these two updates (here and below) are the only place weight was exposed?

Reply:
Yup.